src/Components/CreateImageWizard/validators.ts

```diff
@@ -173,6 +174,51 @@ export const getDuplicateNames = (vg: VolumeGroupWithExtendedLV): string[] => {
   return duplicates;
 };
 
+export const isMountpointValid = (partition: DiskPartition) => {
+  if ('mountpoint' in partition && partition.mountpoint) {
+    for (const [mountpointPath, policy] of Object.entries(MountpointPolicies)) {
+      if (policy.Exact) {
+        if (partition.mountpoint === mountpointPath) {
+          return true;
+        } else if (partition.mountpoint.startsWith(mountpointPath)) {
```

Comment:
I'm struggling to parse this if/else branch. Since we're looking for an exact match could we not just do:
```javascript
return partition.mountpoint === mountpointPath
```

Or am I missing something?